src/v/pandaproxy/parsing/httpd.h

```diff
@@ -36,6 +44,39 @@ T parse_param(std::string_view type, std::string_view key, ss::sstring value) {
     return res.value();
 }
 
+inline ppj::serialization_format parse_serialization_format(
+  std::string_view format,
+  const std::vector<json::serialization_format>& supported) {
+    std::vector<std::string_view> none = {"", "*/*"};
```

Comment:
static constexpr std::array ?

Reply:
could be good to have a quick c++ test for this in addition to the panda proxy for fast iteration cycles.

Reply:
> static constexpr std::array ?

Done

> could be good to have a quick c++ test for this in addition to the panda proxy for fast iteration cycles.

Done